logs.py

```diff
@@ -0,0 +1,17 @@
+import re
+def CheckLog():
+    while True:					
+        searchString = input("Enter search string: ")
+        matchedLines = ""
+        logFile = open("Logs.txt")
+        logFileData = logFile.read().split("\n")
+        logFile.close()
+        for line in logFileData:
+            if re.search(searchString,line,re.IGNORECASE):
```

Comment:
This will do a basic search if I know exactly what I was looking for. However, I can't do any exclusions like "give me all firefox traffic hitting a specific file but ignore this ip"